db/employmentDb.sql

```diff
@@ -1,51 +1,64 @@
-
 CREATE TABLE employee (
-
 	employee_id serial primary key,
 	employee_firstName text not null,
 	employee_lastName text not null,
 	employee_phoneNumber text not null,
 	employee_email text not null,
 	employee_address text not null,
+	employee_role text not null,
 	employee_dev_level text not null,
 	employee_hire_date DATE not null,
 	employee_onLeave BOOLEAN not null,
-	employee_gender text not null
-);
-
-CREATE TABLE project (
-
-	project_id serial primary key,
-	project_name text not null,
-	project_description text not null,
-	project_start_date DATE not null,
-	project_date_date DATE not null
+	employee_gender text not null,
+	employee_status text not null
 );
 
-CREATE TABLE tech (
-
-	tech_id serial primary key,
-	tech_name text not null
-);
-
-CREATE TABLE assignedproject (
-
-	assignedproject_id serial primary key,
+CREATE TABLE assignproject (
+    assignproject_id serial primary key,
 	employee_id INTEGER,
-	project_id INTEGER
+	project_id INTEGER,
```

Comment:
what does this have to do with holiday calendars?!

Reply:
yeah It has nothing to do with it. its for tpms :)